build-config/resources/Ballerina.toml

```
@@ -0,0 +1,15 @@
+[package]
+org = "nuvindu_dias"
+name = "pubsub"
+version = "0.1.0"
```

This should have a placeholder to replace the version at the build time.
```suggestion
version = "@toml.version@"
```